src/AppInstallerRepositoryCore/Microsoft/PredefinedInstalledSourceFactory.cpp

```diff
@@ -121,16 +133,29 @@ namespace AppInstaller::Repository::Microsoft
                 SQLiteIndex index = SQLiteIndex::CreateNew(SQLITE_MEMORY_DB_CONNECTION_TARGET, Schema::Version::Latest());
 
                 // Put installed packages into the index
-                if (filter == PredefinedInstalledSourceFactory::Filter::None || filter == PredefinedInstalledSourceFactory::Filter::ARP)
+                if (filter == PredefinedInstalledSourceFactory::Filter::None || filter == PredefinedInstalledSourceFactory::Filter::ARP ||
+                    filter == PredefinedInstalledSourceFactory::Filter::User || filter == PredefinedInstalledSourceFactory::Filter::Machine)
                 {

```

Comment:
Is this done on purpose for readability? Could be simplified to: 

if (filter != PredefinedInstalledSourceFactory::Filter::Msix)

Reply:
I think it'd be better to check expected values instead of "!=" so future additions of the enum will not accidentally change previous behavior